frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -71,16 +56,24 @@ const AdminJobsPage = () => {
                         <TestJobForm submitAction={submitTestJob} />
                     </Accordion.Body>
                 </Accordion.Item>
```

Comment:
The preferred refactor would be to get rid of lines 67 through 73 of the original code and instead put another item in the jobLaunchers array that looks like this:

```
{
   name: "Test Job"
   form: <TestJobForm submitAction={submitTestJob} />
}

```

You are refactoring in the wrong direction, take short dry code,  and making it long and repetitive.
Please don't do that.